Singer.API/ClientApp/src/app/modules/admin/components/singerevents/singerevent-overview/singerevent-overview.component.html

```diff
@@ -31,9 +31,7 @@
 
       <!-- AgeGroups Column -->
       <ng-container matColumnDef="ageGroups">
-         <th mat-header-cell *matHeaderCellDef>
-            Leeftijds Groepen
-         </th>
+         <th mat-header-cell *matHeaderCellDef>Leeftijds Groepen</th>
```

Comment:
Not really related to this ticket, but I do think this should be `Leeftijdsgroepen`